music21/analysis/neoRiemannian.py

```diff
@@ -681,6 +682,18 @@ def testNeoRiemannianCombinations(self):
         c8_T = LRP_combinations(c8, 'LP')
         self.assertEqual(str(c8_T), '<music21.chord.Chord B-3 E-4 G4>')
 
+        ds_min = chord.Chord('B- D# F#')
+        self.assertEqual(ds_min, LRP_combinations(ds_min, "LP", raiseException=False))
+        with self.assertRaises(LRPException):
+            LRP_combinations(ds_min, "LP", raiseException=True)
+
+        es_min = chord.Chord('B# E# G#')
+        f_min_spelled_diff = chord.Chord('B# F G#')
+        LRP_es_min = LRP_combinations(es_min, "LP", raiseException=True)
```

Comment:
That style guideline is required.  I like `es_min_transformed` over `es_min_LRP` because it's an LP not LRP transformation.

(also please use single quotes entirely)

Reply:
line length can go up to 100 (but I feel you-- 80 is generally better), but I would say good variable names are the highest priority, so if you need to go to another line, that's fine. Do it with an open parenthesis like you've done here instead of a line continuation character.

Reply:
definitely split lines into two.  Though in a doctest, you'll need to replace `>>> ` with `... ` on the second line.  Thanks for the great work.

Music21 style forbids line continuation backslashes.  Maybe soon with editors configured to strip trailing spaces, we can relax that, but in the past the `\ `  with space at the end of the line was the source of no end of hard to diagnose errors.

Reply:
I took that into account in my last commit, I'll let you confirm that it's ok.